aict-MERN/aict-site-w-react/src/components/Main/components/createPostBtn/CreatePostBtn.js

```diff
@@ -0,0 +1,38 @@
+import './createPostBtn.css'
+import { useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faCirclePlus, faPlus } from '@fortawesome/free-solid-svg-icons'
+
+library.add(faCirclePlus, faPlus)
+
+
+const CreatePostBtn = ({setMakeAPostVisible, makeAPostVisible}) => {
+
+    const [style, setStyle] = useState('none')
+
+    const handleOnBtnEnter = () => {
+        setStyle('block')
```

Comment:
you should rename the `style` variable to `isMouseOverComponent` and then derive the actual style needed either before the final return block, or inside it with a ternary (the `a ? b : c` syntax from earlier)